demos/forward.py

```diff
@@ -45,31 +46,75 @@
 
 g_verbose = True
 
+class ForwardServer(SocketServer.ThreadingTCPServer,object):
```

Comment:
It is needed for super() to work as ThreadingTCPServer is an "old style" class. See this thread: http://stackoverflow.com/questions/9698614/super-raises-typeerror-must-be-type-not-classobj-for-new-style-class